lib/db/schema/patch-010-011.sql

```diff
@@ -0,0 +1,11 @@
+-- Add checkPassword stored procedure
+
+CREATE PROCEDURE `checkPassword_2` (
+    IN `inUid` BINARY(16),
```

Comment:
Could this be `checkPassword_1`? I guess you still had a version 1 in your old patch, but we don't yet have on v1 in master so best to start there.